File: {{ cookiecutter.app_name }}/{{ cookiecutter.lib_name }}/src/main/java/{{ cookiecutter.lib_package_dir }}/Module.java
package {{ cookiecutter.lib_package_name }};

/**
 *
 * @author {{ cookiecutter.author_name }}
 * @version {{ cookiecutter.version_package_name }} {% now 'local', '%d/%m/%Y' %}
 * @since {{ cookiecutter.version_package_name }}
 */
public interface Module {
    void sendData(int hash, DataResponse dataResponse );
}

File: {{ cookiecutter.app_name }}/{{ cookiecutter.lib_name }}/src/main/java/{{ cookiecutter.lib_package_dir }}/Module.kt
package {{ cookiecutter.lib_package_name }};

/**
 *
 * @author {{ cookiecutter.author_name }}
 * @version {{ cookiecutter.version_package_name }} {% now 'local', '%d/%m/%Y' %}
 * @since {{ cookiecutter.version_package_name }}
 */
interface Module {
    fun sendData(hash: Int, dataResponse: DataResponse?)
}
